opentelemetry-sdk/src/trace/sampler.rs

```diff
@@ -46,6 +46,13 @@ use opentelemetry_api::{
 };
 use std::convert::TryInto;
 
+mod jaeger_remote;
```

Comment:
Hm could we have this as a feature in `opentelemetry-jaeger`? Just need it to implement [`ShouldSample`](https://docs.rs/opentelemetry/latest/opentelemetry/sdk/trace/trait.ShouldSample.html) and it could be set the same way as the rest of the sampler config

Reply:
I thought about it but decided to leave it here for two reasons.

- Although the name is jaeger remote sampler, the OTLP collector also supports this
- `Sampler` contains all SDK defined sampler and jaeger remote sampler is also one of them.
- We may need to support it in [`OTEL_TRACERS_SAMPLER`](https://sourcegraph.com/github.com/open-telemetry/opentelemetry-rust@HEAD/-/blob/opentelemetry-sdk/src/trace/config.rs) env var.

Reply:
Yeah good point, I missed that this is defined in the sdk spec

Reply:
think it should be an opt-in feature? it does add a few dependencies other than that looks good

Reply:
Yeah I think we can make it opt-in

Reply:
Updated